src/styles/blocks/product.scss

```
@@ -0,0 +1,79 @@
+.product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.product__title {
+  margin: 0 0 4px;
```

just use margin-bottom instead. For similar cases as well
```suggestion
  margin: 4px;
```